src/styles/components/_steps-box.scss

```diff
@@ -0,0 +1,68 @@
+.steps-box{
+    max-width: 21.25rem;
+    margin: auto;
+    margin-bottom: 1.75rem;
+    @include media('tablet') {
+        margin: .3125rem;
+    }
+    @media screen and (min-width:1100px) {
+        margin: 0;
+    }
+
+    &__title{
+        font-family: 'Roboto';
+        font-style: normal;
```

Comment:
I don't think we need this since the default font is Roboto. Maybe the font-style we do, please double check that.